app/src/main/java/com/standuptracker/ui/gallery/GalleryViewModel.kt

```diff
@@ -9,13 +9,13 @@ import com.standuptracker.service.PersonService
 class GalleryViewModel : ViewModel() {
 
     var people: MutableLiveData<ArrayList<Person>> = MutableLiveData<ArrayList<Person>>()
-    var personService: PersonService = PersonService()
+    private var personService: PersonService = PersonService()
```

Comment:
It is best practice to keep each variable at its least access necessary, and here the person service is only used in this view model

Reply:
yes, agreed.